capstone/src/main/java/com/google/sps/servlets/Login.java

```diff
@@ -13,19 +13,7 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html");
-
     UserService userService = UserServiceFactory.getUserService();
-    String logMsg = "";
-    String redirectUrl = "/";
-
-    if (!userService.isUserLoggedIn()) {
-      logMsg = "user is now logged in";
-      redirectUrl = userService.createLoginURL("/login");
-    } else {
-      logMsg = "user is already logged in";
-      redirectUrl = "statistics.html";
-    }
-    response.getWriter().println(logMsg);
-    response.sendRedirect(redirectUrl);
+    response.sendRedirect(userService.createLoginURL("statistics.html"));
```

Comment:
It seems like you assume that requests to this server will only be made if the user is logged in. Can you add an error status if they are logged in already? Is some sort of error handling behavior already built in to the create login url method?

Reply:
This server is only called when a user is not logged in. This is checked with a separate sever /status in the showStats() function. For this reason I do not need error handling behavior in that file.

Reply:
Even if you chose not to call it when the user is not logged in, if I were someone messing around I could still make a request from the console to this servlet. Its not strictly necessary, but I think it would be nice to have the check in the servlet as well.